content/api/cypress-api/keyboard-api.md

```diff
@@ -44,8 +44,8 @@ Cypress.Keyboard.defaults({
 ### Where to put Keyboard configuration
 
 A great place to put this configuration is in your
-[`cypress/support/index.js` file](/guides/core-concepts/writing-and-organizing-tests#Support-file),
-since it is loaded before any test files are evaluated.
+[`cypress/support/e2e.js` or `cypress/support/component.js`](guides/core-concepts/writing-and-organizing-tests#Support-file)
```

Comment:
@cowboy let's sync up on if this is still the best place to link users to. if it is I can wrap the other refs to point here as well.

Reply:
We're now linking these refs to `/guides/core-concepts/writing-and-organizing-tests#support-file`